server/game/cards/plots/06/duel.js

```diff
@@ -0,0 +1,65 @@
+const PlotCard = require('../../../plotcard.js');
+
+class Duel extends PlotCard {
+    setupCardAbilities() {
+        this.whenRevealed({
+            handler: () => {
+                let opponent = this.game.getOtherPlayer(this.controller);
+                if(!opponent) {
+                    return;
+                }
+
+                if(this.notEnoughTargets()) {
+                    return;
+                }
+
+                this.game.addMessage('{0} uses {1} to have {2} choose 2 non-Army characters with printed cost 6 or higher', 
+                                      this.controller, this, opponent);
+
+                this.game.promptForSelect(opponent, {
+                    numCards: 2,
+                    activePromptTitle: 'Select two characters',
+                    source: this,
+                    cardCondition: card => !card.hasTrait('Army') && card.getType() === 'character' && card.getCost() >= 6,
+                    onSelect: (player, cards) => this.targetsSelected(player, cards)
+                });
+            }
+        });
+    }
+
+    targetsSelected(player, cards) {
+        this.targets = cards;
```

Comment:
Select card prompts work by selecting 'up to 2 cards' by default. So you'll need to check that exactly 2 cards were selected here, and return false if they didn't.